tools/server-admin/build.gradle.kts

```diff
@@ -145,3 +150,12 @@ if (Os.isFamily(Os.FAMILY_WINDOWS)) {
 if (Os.isFamily(Os.FAMILY_MAC) && System.getenv("CI") != null) {
   tasks.named<Test>("intTest").configure { this.enabled = false }
 }
+
+tasks.named<Test>("intTest").configure {
+  // Reduce likelihood of OOM due to too many Quarkus apps in memory;
+  // Ideally, set this number to the number of IT classes to run for each backend.
+  forkEvery = 4
```

Comment:
Better reduce to `1` (safer ;)

Reply:
With 1, the overall build becomes quite slow because a new container is started for every combination of test class + backend. With 4, a new container is spawned just when the backend changes.

Reply:
Okay - then let's go with 4 for now.

However, I suspect there's a classloader/resource leak related to tests.

Reply:
> However, I suspect there's a classloader/resource leak related to tests.

Can be investigated separately.

Reply:
https://github.com/projectnessie/nessie/issues/8571